paimon-web-ui-new/src/views/metadata/components/menu-tree/index.tsx

```diff
@@ -120,7 +120,7 @@ export default defineComponent({
             <n-space justify='space-between' align='enter'>
               <article>Catalog</article>
               <n-button size='small' quaternary circle>
-                <n-icon>
+                <n-icon >
```

Comment:
What's this ?

Reply:
I added it by accident. It's been deleted